src/Drum.java

```diff
@@ -5,14 +5,14 @@
 
     private ArrayList<Ball> allBalls = new ArrayList<>();
     private Random generator = new Random();
+    private final int numberOfBalls = 45;
 
     public Drum() {
     }
 
     protected void collectAllBalls(){
-        for (int i = 0; i < 45; i++){
-            int ballNumber = i + 1;
-            allBalls.add(new Ball(ballNumber));
+        for (int i = 1; i < numberOfBalls + 1; i++){
```

Comment:
I should change this to `for (int i = 1; i <= numberOfBalls;  i++)` it uses less magic numbers (+1).